crates/transaction-emitter-lib/src/cluster.rs

```diff
@@ -152,7 +152,7 @@ impl Cluster {
     ) -> Result<LocalAccount> {
         let sequence_number = query_sequence_number(client, address).await.map_err(|e| {
             format_err!(
-                "query_sequence_numbers on {:?} for account {} failed: {}",
+                "query_sequence_numbers on {:?} for account {} failed: {:?}",
```

Comment:
We could consider using `{:#}` as well if you want to see the standard "context stack" of an anyhow error.

Reply:
{:?} seems to be strictly showing more:

https://docs.rs/anyhow/latest/anyhow/struct.Error.html